src/styles/global.css

```
@@ -36,43 +36,53 @@
+    --maple: #e11d48;
+    --line: rgba(247, 244, 237, 0.22);
+    --line-subtle: rgba(247, 244, 237, 0.12);
+    --glass-border: rgba(247, 244, 237, 0.16);
+    --radius-sm: 0.6rem;
+    --radius-md: 1rem;
+    --radius-lg: 1.6rem;
+    --shadow-sm: 0 0.5rem 1.5rem rgba(0, 0, 0, 0.3);
+    --shadow: 0 2rem 5rem rgba(0, 0, 0, 0.45);
+    --shadow-glow: 0 1rem 3rem rgba(16, 185, 129, 0.15);
+    --display: "Iowan Old Style", "Baskerville", "Palatino Linotype", "Georgia", serif;
+    --sans: "Avenir Next", "Segoe UI", -apple-system, BlinkMacSystemFont, "Helvetica Neue", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)